examples/README.md

```diff
@@ -40,8 +40,8 @@ The processing will be kicked of by sending a request to the HTTP component. The
 
 - receives a raw http request
 - returns the 7th minute unix interval for the current server time
-- first makes a request to seventh
-- seventh responds with the timing information
+- http makes a request to the timing service
```

Comment:
the test here is for the server cache. Maybe better to try use this term to connect conceptually with the related feature. Timing service is just a naive implementation of the server side cache

Reply:
`timing` is a very bad name indeed. Would you suggest `cache`, `server-cache`?

Reply:
`http-cache` maybe?

Reply:
`http-cache` makes it super obvious, I like it